### PR TITLE
Write to file

### PR DESCRIPTION
**Pull request by indigo-sadland**
Added cmd option that allows to save output to a file.

### CODE REVIEW

File: internal/executor/executor.go [hunk 1]
```diff
@@ -1,24 +1,51 @@
 package executor
 
 import (
+	"bufio"
 	"fmt"
+	"github.com/projectdiscovery/gologger"
+	"os"
+	"sync"
```
Comment (dwisiswant0) [inline internal/executor/executor.go:8]: Revert the order of importing packages like the previous one.

File: internal/executor/executor.go [hunk 2]
```diff
@@ -1,24 +1,51 @@
...
+}
+
 // New to execute target hostname
 func New(opt *option.Options, hostname string) {
 	var out = ""
 
 	vuln, DNS, err := exec(hostname)
 	if err != nil {
-		gologger.Error().Msgf("%s: %s", hostname, err.Error())
+		out += fmt.Sprintf("[%s] %s: %s", aurora.Red("ERR"), hostname, err.Error())
```
Comment (dwisiswant0) [inline internal/executor/executor.go:48]: DO NOT save the error to a file, leave it on `stderr` with `gologger.Error()`.

File: internal/option/const.go [hunk 3]
```diff
@@ -18,10 +18,12 @@ const (
   -t, --target <HOST/FILE>    Define single target host/list to check
   -c, --concurrent <i>        Set the concurrency level (default: 25)
   -s, --silent                Suppress errors and/or clean output
+  -o, --output                Define path where you want to store the tool's output
```
Comment (dwisiswant0) [inline internal/option/const.go:21]:
```diff
-  -o, --output                Define path where you want to store the tool's output
+  -o, --output <FILE>         Save vulnerable hosts to FILE
```

File: internal/option/options.go [hunk 4]
```diff
@@ -2,8 +2,9 @@ package option
...
 type Options struct {
-	Target      string
-	Concurrency int
-	Silent      bool
-	List        []string
+	Target      	string
+	Concurrency 	int
+	Silent      	bool
+	List        	[]string
+	Output			string
```
Comment (dwisiswant0) [inline internal/option/options.go:9]: Use space indentation instead of tabs.

**Review by dwisiswant0 — CHANGES_REQUESTED** [review]

**Review by indigo-sadland — COMMENTED** [review]
I hope now all is done in the proper way!

File: internal/executor/executor.go [hunk 5]
```diff
@@ -12,6 +15,30 @@ import (
 	"github.com/pwnesia/dnstake/pkg/fingerprint"
 )
 
+var mu sync.Mutex
+
+// WriteToFile writes output data into specified file.
+func WriteToFile(data, outputFile string) {
+
```
Comment (dwisiswant0) [inline internal/executor/executor.go:22]: Remove this trailing newline.

**Review by dwisiswant0 — APPROVED** [review]
Thanks for your contributions, @indigo-sadland! Merging now.

**Pull request merged**